packages/react-components/src/templates/__tests__/SharedResearchOutput.test.tsx

```diff
@@ -1025,3 +1024,218 @@ describe('the switch to draft button', () => {
     });
   });
 });
+
+describe('the publish button', () => {
+  it('does not render if the user is not staff', () => {
+    const { queryByText } = render(
+      <ResearchOutputPermissionsContext.Provider
+        value={{
+          canPublishResearchOutput: false,
+        }}
+      >
+        <SharedResearchOutput
+          {...props}
+          documentType="Article"
+          published={false}
+          reviewRequestedBy={{
+            id: 'user-1',
+            firstName: 'User',
+            lastName: 'One',
+          }}
+        />
+        ,
+      </ResearchOutputPermissionsContext.Provider>,
+    );
+    expect(queryByText('Publish')).not.toBeInTheDocument();
```

Comment:
Shouldn't you make this test also show that a publish button is displayed when you are staff?